### ISSUE TITLE
On the distance between two neural networks and the stability of learning

### ISSUE THREAD

**Issue by nocotan** [CompML/stochastic-optimization-survey#10]
## 一言でいうと
ニューラルネットワークのパラメータ間の距離と最適化の安定性の関係について分析．

### 論文リンク
https://arxiv.org/pdf/2002.03432.pdf

### 著者/所属機関
J Bernstein (Caltech), A Vahdat (NVIDIA), Y Yue Caltech, MY Liu (NVIDIA)

### 投稿日付(yyyy/MM/dd)
2020/06/12

## 概要

<img width="720" alt="Screen Shot 2021-01-02 at 14 37 36" src="https://user-images.githubusercontent.com/10952293/103451503-1312ac00-4d08-11eb-8f49-b983a5d2e177.png">

ニューラルネットワークの勾配降下法における二次罰則の項を，ニューラルネットの構造に合わせた新しい距離関数で置き換える：

* ニューラルネットワークとヤコビアンの摂動解析から，新しい距離関数であるdeep relative trustを導出．
* この距離関数から導出される補題から，新しい学習アルゴリズムであるFrobenius matched gradient descent (Fromage)を導出． 
* 多数のタスクにおいて， Fromageは学習率チューニングなしに良好な性能を達成可能．

## 新規性・差分
勾配法とパラメータの関係についての分析から，ニューラルネットワークのより深い理解につながる．

## 手法

<img width="817" alt="Screen Shot 2021-01-02 at 14 33 31" src="https://user-images.githubusercontent.com/10952293/103451444-823bd080-4d07-11eb-9d9b-d4404188402b.png">

<img width="823" alt="Screen Shot 2021-01-02 at 14 35 07" src="https://user-images.githubusercontent.com/10952293/103451469-badbaa00-4d07-11eb-916a-74ba094cc935.png">

## 結果

<img width="1096" alt="Screen Shot 2021-01-02 at 14 34 16" src="https://user-images.githubusercontent.com/10952293/103451457-9bdd1800-4d07-11eb-9faf-61f152001591.png">

<img width="806" alt="Screen Shot 2021-01-02 at 14 34 49" src="https://user-images.githubusercontent.com/10952293/103451464-af887e80-4d07-11eb-905d-fbb26c8e147a.png">
